src/libponyc/pass/refer.c

```diff
@@ -1167,6 +1167,11 @@ static bool refer_consume(pass_opt_t* opt, ast_t* ast)
           return false;
         }
       }
+      else if (ast_id(left) == TK_CALL) {
+          ast_error(opt->check.errors, ast,
+              "Consume expressions must specify a single identifier");
+          return false;
+      }
```

Comment:
Two suggested changes here, for consistency purposes:

- Fixing indentation and C line break style to match other code
- Using lowercase to start the error message, as other compile errors do

```suggestion
      } else if (ast_id(left) == TK_CALL) {
        ast_error(opt->check.errors, ast,
          "consume expressions must specify a single identifier");
        return false;
      }
```